content/posts/2015-01-16-open-source-for-good-government.md

```diff
@@ -2,7 +2,6 @@
 title: Open source for good government
 date: '2015-01-16'
 layout: post
-image: /assets/blog/join-us/18f_govuk.png
```

Comment:
Keep (see other comment)

Reply:
Restored in https://github.com/18F/18f.gsa.gov/pull/3863/commits/ef47484d22fc375a653dac756676b07b69149b86#diff-07442e76559173210cb3172cbb544e5ef6b3747d7006186cd00730ae54ff7f23R84